src/backend/aspen/api/views/users.py

```diff
@@ -1,19 +1,16 @@
-from fastapi import APIRouter, Depends, HTTPException
+from fastapi import APIRouter, Depends
 from sqlalchemy.ext.asyncio import AsyncSession
 from starlette.requests import Request
 
+from aspen.api.auth import get_auth_user
 from aspen.api.deps import get_db
-from aspen.api.schemas.users import User as userschema
-from aspen.database.models.usergroup import User
+from aspen.api.schemas.users import UserMeResponse
 
 router = APIRouter()
 
 
-@router.get("/me", response_model=userschema)
+@router.get("/me", response_model=UserMeResponse)
```

Comment:
We're trying to keep the places where we expose split ID's to a minimum -- for now it's only on the /users/me endpoint.